server/app.js

```diff
@@ -57,8 +57,6 @@ const middleware = [
 
 app.get('/__gtg', (req, res) => res.sendStatus(200));
 
-app.get('/syndication/__gtg', (req, res) => res.sendStatus(200));
```

Comment:
it’s possible some healthchecks depend on this but we can probably wait until it’s merged to find out

Reply:
Health checks seem ok